packages/jaeger-ui/src/model/ddg/transform-ddg-data.tsx

```diff
@@ -12,64 +12,65 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import {
-  TDdgPayload,
-  TDdgService,
-  TDdgOperation,
-  TDdgPath,
-  TDdgServiceMap,
-  TDdgPathElemsByDistance,
-  TDdgTransformedDdgData,
-  PathElem,
-} from './types';
+import { PathElem, TDdgModel, TDdgPayload, TDdgPath, TDdgPathElemsByDistance, TDdgServiceMap } from './types';
 
 export default function transformDdgData(
```

Comment:
Since we moved to the naming convention of filenames reflecting the default export, the file should be named `transformDdgData.tsx`. Unfortunately, that was not the initial approach taken, so it's not reflected, everywhere.

Reply:
I see, I will rename. The usage of `kebab-case` vs `PascalCase` vs `camelCase` file names definitely seem inconsistent. It might be worth mentioning the style guide portion of `CONTRIBUTING.md`.

Reply:
It probably is inconsistent due to not renaming everything when adopting the approach... The approach is to name based on default export, if there is one. If not, then a `descriptive-name` kebab-case name.

Reply:
Yeah, we need to put a bit of time into a style-guide. I started some notes, here:

https://github.com/jaegertracing/jaeger-ui/blob/style-guide-wip/CONTRIBUTING.md

Reply:
Created a ticket for it: https://github.com/jaegertracing/jaeger-ui/issues/385